account_test.go

```diff
@@ -47,7 +47,104 @@ func TestAccountUnmarshal(t *testing.T) {
 				},
 			},
 		},
+		"legal_entity": map[string]interface{}{
+			"additional_owners": []map[string]interface{}{
+				{
+					"address": map[string]interface{}{
+						"city":        "city1",
+						"country":     "US",
+						"line1":       "line1",
+						"line2":       "line2",
+						"postal_code": "90210",
+						"state":       "CA",
+					},
+					"dob": map[string]interface{}{
+						"day":   1,
+						"month": 1,
+						"year":  1970,
+					},
+					"first_name":                  "First",
+					"last_name":                   "Last",
+					"maiden_name":                 "Maiden",
+					"personal_id_number_provided": true,
+					"verification": map[string]interface{}{
+						"details":       "details",
+						"details_code":  "failed_other",
+						"document":      "file_123",
+						"document_back": "file_234",
+						"status":        "pending",
+					},
+				},
+				{
+					"address": map[string]interface{}{
+						"city": "city2",
+					},
+					"dob": map[string]interface{}{
+						"day":   1,
+						"month": 1,
+						"year":  1960,
+					},
+					"first_name": "First2",
+				},
+			},
+			"address": map[string]interface{}{
+				"city":        "city-3",
+				"country":     "US",
+				"line1":       "line1-3",
+				"line2":       "line2-3",
+				"postal_code": "91111",
+				"state":       "CA",
+			},
+			"business_tax_id_provided": true,
+			"dob": map[string]interface{}{
+				"day":   1,
+				"month": 1,
+				"year":  1980,
+			},
+			"first_name":  "John",
+			"last_name":   "Doe",
+			"maiden_name": "Maiden",
+			"personal_address": map[string]interface{}{
+				"city":        "personal_city1",
+				"country":     "US",
+				"line1":       "personal_line1",
+				"line2":       "personal_line2",
+				"postal_code": "90210",
+				"state":       "CA",
+			},
+			"personal_id_number_provided": true,
+			"ssn_last_4_provided":         true,
+			"type":                        "company",
+			"verification": map[string]interface{}{
+				"details":       "details",
+				"details_code":  "failed_other",
+				"document":      "file_345",
+				"document_back": "file_456",
+				"status":        "unverified",
+			},
+		},
+		"metadata": map[string]interface{}{
+			"key1": "value1",
+			"key2": "value2",
+		},
+		"payout_schedule": map[string]interface{}{
+			"delay_days": 2,
+			"interval":   "weekly",
+		},
+		"tos_acceptance": map[string]interface{}{
+			"date":       1528573382,
+			"ip":         "127.0.0.1",
+			"user_agent": "user agent",
+		},
 		"type": "custom",
+		"verification": map[string]interface{}{
+			"disabled_reason": "fields_needed",
+			"due_by":          1528573382,
+			"fields_needed": []interface{}{
```

Comment:
You have no idea how long I spent on figuring out the write combination of `[]` `string` `map` `interface` and `{}` to make that one work 😓

Reply:
Oh god, haha.